keyboards/cheshire/curiosity/keymaps/via/keymap.c

```diff
@@ -0,0 +1,56 @@
+/*
```

Comment:
This should be in a separate PR, ideally.

Reply:
I'm hoping this can proceed for now. will remember it for next time